pom.xml

```diff
@@ -149,7 +149,7 @@
     <avro.mapred.classifier>hadoop2</avro.mapred.classifier>
     <aws.kinesis.client.version>1.12.0</aws.kinesis.client.version>
     <!-- Should be consistent with Kinesis client dependency -->
-    <aws.java.sdk.version>1.11.271</aws.java.sdk.version>
```

Comment:
Should we add a comment reminding to change `aws.java.sdk.version` too in front of `aws.kinesis.client.version`?

Reply:
Thanks. For now, `<!-- Should be consistent with Kinesis client dependency -->` at line 151 looks enough to me. Although one line ahead (`in front of`) sounds better than one line below, the warning content will be the same.